### DISCUSSION TITLE
Problem with adding JS to my html sheet (via the basic jsDelivr CDN link) - What am I doing wrong??

### DISCUSSION THREAD

**Discussion by site-dever** [twbs/bootstrap#34312] [General]
I was told to put it just before the closed body tag. 

There was also the bundle link that didn't work:

Everything works (the jsdelivr CDN for CSS loaded up fine in the HEADER).

But whenever I do the simple JS functions (with the data toggles) they just don't work at all.

- I'm using a mac, and used various browsers, all the same result.

Here's the HTML top side screenshot: https://paste.pics/c5ceb923e80364057f7c80da3e0a7350
Here's the HTML bottom screenshot: https://paste.pics/77d881e6cfa8458eb2dd33a4a989b4fa

Thank you so much.

**Comment by coliff** [comment]
You should post a Codepen or similar so that others can help identify the issue.
Have you checked the browser dev tools? That will show in console if there is a JS error.
